main.go

```diff
@@ -95,14 +113,116 @@ func enableCors(w *http.ResponseWriter) {
 
 func handleRequest() {
 
+	serverAddr, ok := os.LookupEnv("JOBS_SERVER_ADDR")
+	if !ok {
+		serverAddr = ":8081"
```

Comment:
this is just opinion, but i think it reads better

```suggestion
	serverAddr, err := os.LookupEnv("JOBS_SERVER_ADDR")
	if err != nil {
		serverAddr = ":8081"
```

Reply:
wait... did i understand that correctly?

Reply:
Just to recap our conversation in Slack my understanding is `ok` is idiomatic Go for `bool` types and err is for `error` types with [`os.LookupEnv`](https://golang.org/pkg/os/#LookupEnv) returning a `string` and `bool` types respectively `ok` still makes sense to me but I'm happy to change.